orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// Graph transformers are not needed for on-device training and can be left
+// out at compile time.
+#ifdef ENABLE_TRAINING
```

Comment:
Do we need this given we are excluding this file in optimizer cmake?

Reply:
We can probably remove it. Will make the change.